packages/components/pie-modal/src/defs.ts

```diff
@@ -105,6 +105,12 @@ export interface ModalProps extends RTLComponentProps {
      * The position of the modal; this controls where it will appear on the page.
      */
     position: typeof positions[number];
+
+    aria?: {
+        close?: string;
+        back?: string;
+        loading?: string;
+    };
```

Comment:
can you make this a type so we don't write it out in multiple places please?

Reply:
Fixed.